spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -27,6 +27,9 @@ This section describes the REST-api, that monitoring agents would use to retriev
 * When using JSON format, the REST API will respond to GET requests with data formatted in a tree like fashion with sub-trees for the sub-resources.
 A sub-tree that does not contain data must be omitted.
 * A 'shadow tree' that responds to OPTIONS will provide the metadata.
+* The application name will be provided as tag `__app=<name>` if <<app-multi-app,multiple applications>> are deployed in the server.
```

Comment:
We should change it to a single underscore `_app=<name>` as Prometheus reserves `__` for internal use.

https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels